lib/Test/Nginx/Util.pm

```diff
@@ -2364,6 +2373,23 @@ retry:
             #warn "pid file not found";
         }
     }
+
+shutdown_error_log:
+    if (defined $block->shutdown_error_log) {
```

Comment:
if `$block->shutdown_error_log` {
    `# shutdown nginx`
    `# CheckShutDownErrorLog`
}

"shutdown" operation in the following, I want to control it in a smaller range.